src/components/surveys/survey-design/question-edit/rhs-subcontrols/YearRadioGroup.tsx

```diff
@@ -0,0 +1,81 @@
+import {SimpleTextInput, SimpleTextLabel} from '@components/widgets/StyledComponents'
+import {Box, FormControl, FormControlLabel, Radio, RadioGroup} from '@mui/material'
+import {theme} from '@style/theme'
+import React from 'react'
+
+export type YearFormatType = 'ANY' | 'CURRENT' | 'SET'
+
+const capitalizeFirstLetter = (val: string) => {
+  return `${val.charAt(0).toUpperCase()}${val.substring(1)}`
+}
+
+type YearRadioGroupProps = {
+  type: 'min' | 'max'
+  yearFormat: YearFormatType
+  yearValue: number | undefined
+  hasError: boolean
+  onChange: (yearFormat: YearFormatType, yearValue: number | undefined) => void
+}
+
+const YearRadioGroup: React.FunctionComponent<YearRadioGroupProps> = ({
```

Comment:
Presentational component that calls `onChange` when a new year format is selected or when a year value is updated.